proto/vlan_state.proto

```diff
@@ -0,0 +1,12 @@
+/*
+ * Port VLAN state
+ */
+syntax = "proto3";
+
+message VLANState {
```

Comment:
Should be VlanState (in this context, 'vlan' is considered a word, so it would be VlanState or vlan_state depending on the language style).

Reply:
Updated. PTAL..